tools/get_config.py

```diff
@@ -36,19 +37,15 @@
 if __name__ == '__main__':
     # Parse Options
     parser = get_default_options_parser(add_clean=False, add_options=False)
-    parser.add_option("--source", dest="source_dir",
-                      default=None, help="The source (input) directory", action="append")
-    parser.add_option("--prefix", dest="prefix", action="append",
-                      default=None, help="Restrict listing to parameters that have this prefix")
-    parser.add_option("-v", "--verbose", action="store_true", dest="verbose",
+    parser.add_argument("--source", dest="source_dir", type=argparse_filestring_type,
+                        default=[], help="The source (input) directory", nargs="*")
+    parser.add_argument("--prefix", dest="prefix", nargs="*",
+                      default=[], help="Restrict listing to parameters that have this prefix")
+    parser.add_argument("-v", "--verbose", action="store_true", dest="verbose",
                       default=False, help="Verbose diagnostic output")
 
-    (options, args) = parser.parse_args()
+    options = parser.parse_args()
 
-    for path in options.source_dir :
-        if not isdir(path) :
-            args_error(parser, "[ERROR] you passed \"{}\" to --source, which does not exist".
-                       format(path))
     # Target
     if options.mcu is None :
```

Comment:
@theotherjimmy I just noticed this line, will this ever be true now that the `--mcu` option is using an `argpars_many` type? Or will it just be an empty array? This line should probably just change to a "falsey" check instead of an explicit check for `None`:

```
if options.mcu:
```